backend/api/EventHandlers/InspectionFindingEventHandler.cs

```diff
@@ -103,54 +103,51 @@ private async Task<List<Finding>> GenerateFindingsList(List<InspectionFinding> i
             return findingsList;
         }
 
-        public static string GenerateReportFromFindingsReportsList(List<Finding> findingsReports)
+        public static string? GenerateAdaptiveCard(string title, int numberOfFindings, List<Finding> findingsReports)
         {
-            var reportBuilder = new StringBuilder("Findings Report:");
-            reportBuilder.Append(Environment.NewLine);
-            string dateFormat = "dd/MM/yyyy HH:mm";
-            var formatProvider = CultureInfo.InvariantCulture;
-            foreach (var finding in findingsReports)
+            var findingsJsonArray = new JArray();
+
+            for (int i = 0; i < findingsReports.Count; i++)
```

Comment:
Could we use foreach(finding in findingsReports)?

Reply:
Done.